### ISSUE TITLE
Support compressed data in fetch node polyfill

### ISSUE THREAD

**Issue by ibgreen** [visgl/loaders.gl#496]
Problem: Currently fetch polyfill under Node cannot deal with compressed content. If server only supports compressed content load will fail. 

Use case: i3s content is compressed

Solution: Node documentation shows how to implement this, see https://nodejs.org/api/zlib.html

Maybe this code could go here: https://github.com/uber-web/loaders.gl/blob/master/modules/polyfills/src/fetch-node/response.node.js

```js
// Client request example
const zlib = require('zlib');
const http = require('http');
const fs = require('fs');
const request = http.get({ host: 'example.com',
                           path: '/',
                           port: 80,
                           headers: { 'Accept-Encoding': 'br,gzip,deflate' } });
request.on('response', (response) => {
  const output = fs.createWriteStream('example.com_index.html');

  switch (response.headers['content-encoding']) {
    case 'br':
      response.pipe(zlib.createBrotliDecompress()).pipe(output);
      break;
    // Or, just use zlib.createUnzip() to handle both of the following cases:
    case 'gzip':
      response.pipe(zlib.createGunzip()).pipe(output);
      break;
    case 'deflate':
      response.pipe(zlib.createInflate()).pipe(output);
      break;
    default:
      response.pipe(output);
      break;
  }
});
```

**Comment by kylebarron** [comment]
Is this fixed as of #743?

**Comment by ibgreen** [comment]
Yes, but not yet published. It is preferable close an open ticket by indicating a version of loaders.gl that implements it (e.g. `2.1.7` or `2.2.0-alpha.1`). 

Let's look into giving you publishing rights.

**Comment by kylebarron** [comment]
Now this is fixed and included in 2.2.0 I believe.
